### PR TITLE
[release/5.0] Merge, update baselines, SDK and runtimes

### PR DESCRIPTION
**Pull request by dougbu**
- dotnet/aspnetcore-internal#3783
- Merge commit 'e882394a7bb38930da748291fe2c2ceaa6a80588' into 'release/5.0'
- Update `$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)`
- Update package baselines
- Update SDK and runtime versions

### CODE REVIEW

File: eng/Baseline.Designer.props [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
   <PropertyGroup Condition=" '$(PackageId)' == 'dotnet-sql-cache' ">
-    <BaselinePackageVersion>5.0.3</BaselinePackageVersion>
+    <BaselinePackageVersion>5.0.4</BaselinePackageVersion>
   </PropertyGroup>
   <!-- Package: Microsoft.AspNetCore.ApiAuthorization.IdentityServer-->
   <PropertyGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.ApiAuthorization.IdentityServer' ">
-    <BaselinePackageVersion>5.0.3</BaselinePackageVersion>
+    <BaselinePackageVersion>5.0.4</BaselinePackageVersion>
   </PropertyGroup>
   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.ApiAuthorization.IdentityServer' AND '$(TargetFramework)' == 'netcoreapp5.0' ">
```
Comment (dougbu) [inline eng/Baseline.Designer.props:23]: These conditions are still wrong. Looks like the baseline generator in 5.0 is not up-to-date with the one in 3.1. I'll fix this as part of this PR.

**Review by wtgodbe — CHANGES_REQUESTED** [review]
Please change `ValidateBasline` to `true`

**Comment by wtgodbe** [comment]
> ##[error]src/Components/WebAssembly/Server/src/Microsoft.AspNetCore.Components.WebAssembly.Server.csproj(0,0): error NU1603: (NETCORE_ENGINEERING_TELEMETRY=Restore) Microsoft.AspNetCore.Components.WebAssembly.Server depends on Microsoft.NETCore.BrowserDebugHost.Transport (>= 5.0.4-servicing.21116.14) but Microsoft.NETCore.BrowserDebugHost.Transport 5.0.4-servicing.21116.14 was not found. An approximate best match of Microsoft.NETCore.BrowserDebugHost.Transport 5.0.5-servicing.21126.7 was resolved.

Thought I had already pushed everything. Looking

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**